ui/plugins/pinboard.js

```diff
@@ -252,6 +252,14 @@ treeherder.controller('PinboardCtrl', [
             $rootScope.$broadcast('blur-this', "classification-comment");
         };
 
+        // The manual bug entry input eats the global ctrl+enter save() shortcut.
+        // Force that event to be emitted so ctrl+enter saves the classification.
+        $scope.ctrlEnterSaves = function(ev) {
+            if (ev.ctrlKey && ev.keyCode === 13) {
```

Comment:
I think you'd be fine either way.  The ``meta`` is the mac command key.  I don't think people would expect that, however, since the user guide doesn't specify it.